Makefile.am

```diff
@@ -58,6 +58,7 @@ fortrandir = $(datadir)/$(PACKAGE_NAME)/fortran/
 dist_fortran_DATA = $(qmckl_f)
 
 AM_CPPFLAGS = -I$(srcdir)/src -I$(srcdir)/include
+AM_FCFLAGS = -fno-underscoring
```

Comment:
Vijay is checking this...

Reply:
Hi @pablooliveira I'm looking into this and will remove it if it is not needed ASAP.

Reply:
Thanks for looking into this @v1j4y .After discussing with @fmgjcoppens we believe it can be removed safely. So we are going to proceed with removal. If you later feel it need to be added back, we'll do that !

Reply:
I will remove it for now and merge the PR since we need SMW tomorrow to start testing it in QMC=Chem. 

If it turns out we need it we can put it back.